website/docs/r/flow_log.html.markdown

```diff
@@ -18,7 +18,7 @@ interface, subnet, or VPC. Logs are sent to a CloudWatch Log Group or a S3 Bucke
 ```hcl
 resource "aws_flow_log" "example" {
   iam_role_arn    = "${aws_iam_role.example.arn}"
-  log_destination = "${aws_cloudwatch_log_group.example.arn}"
+  log_destination = "${aws_cloudwatch_log_group.test_role.arn}"
```

Comment:
Hi @erikkn 👋 It looks like this was accidentally changed as well -- the `aws_cloudwatch_log_group` resource name is still `example` on line 26:

```hcl
resource "aws_cloudwatch_log_group" "example" {
```

If you can revert this, we can get this in, thanks.

Reply:
```suggestion
  log_destination = "${aws_cloudwatch_log_group.example.arn}"
```